client/src/app/modules/alta-main/alta-main.module.ts

```diff
@@ -1,15 +1,26 @@
 import { NgModule } from '@angular/core';
 import { CommonModule } from '@angular/common';
-import { FormsModule, ReactiveFormsModule } from '@angular/forms';
+import { ReactiveFormsModule } from '@angular/forms';
 import { HttpClientModule } from '@angular/common/http';
 import { RouterModule } from '@angular/router';
 import { HomeComponent } from 'src/app/components/home/home.component';
 import { AuthService } from 'src/app/services/auth.service';
 import { DashboardComponent } from 'src/app/components/dashboard/dashboard.component';
+import { SideNavComponent } from 'src/app/components/sidenav/sidenav.component';
+import { MatModule } from '../material/material-module';
 
 @NgModule({
-  declarations: [HomeComponent, DashboardComponent],
-  imports: [CommonModule, ReactiveFormsModule, HttpClientModule, RouterModule],
+  declarations: [
+    HomeComponent,
+    DashboardComponent,
+    SideNavComponent],
+  imports: [
+    CommonModule,
+    MatModule,
+    ReactiveFormsModule,
+    HttpClientModule,
+    RouterModule
```

Comment:
You forgot to add inside your imports the angular material-specific modules that you are using. I would say do not import the whole module library, just do it by a specific module(like I did in the auth module). Just check if its better for performance to just add it by subcomponents that you use.
<img width="596" alt="Screen Shot 2020-10-11 at 7 09 14 PM" src="https://user-images.githubusercontent.com/35696935/95692647-435d1980-0bf5-11eb-816e-40dee250c820.png">

Reply:
I wanted to import all the mat modules in one place for simplicity. If the module is big it will effect build time not performance. You think the module will get big enough for it to matter ?

Reply:
So I've looked a bit into it, I think since we are going to use angular material all across the website, we should probably create a specific module that will just be responsible of importing the particular components of angular material that we are using, and then we would be able to import this particular module inside the other modules that need it. So this way it would make the code more organized.
https://stackoverflow.com/questions/57038161/angular-8-is-importing-all-material-modules-into-the-main-app-module-better-th

Reply:
Yes exactly. That was my reasoning. The module is indeed MatModule